bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +98,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
I assume that
```
body = stdout[:stdout.find('\n')] 
```
is a safe move.
Because that's essentially what's being done [here](https://github.com/coala/coala-bears/blob/master/bears/vcs/git/GitCommitBear.py#L76-L81)